app/modules/job_control/models.py

```diff
@@ -2,10 +2,6 @@
 """Provides UI space served directly from this application"""
 import logging
 
-from app.modules.asset_groups.models import AssetGroupSighting
```

Comment:
I understood that the problem that having the imports ithin functiosn was now fixed and they could be at the module level. Why did you move this?